sdk/va-report-components/documentation/docs/api/ObjectHandle.md

```diff
@@ -38,3 +38,43 @@ If no `options` parameter is supplied, the report is exported using the default
 
 Refreshes the data for the report object that is controlled by the
 `ObjectHandle`.
+
+### getSelectedData(options?: {formatData: boolean | "datesOnly"}): ReportObjectResultData[]
+
+Returns a user's selection data from the visual. Returns an empty array if the visual has no selections.
+
+#### Arguments
+
+`options` is an optional options bundle for customizing the returned data. The following options are supported:
+
+- `formatData` for specifying the format of the returned data. It can be passed the following values:
+  - `true` to receive all formatted data
+  - `false` to receive all unformatted data
+  - `"datesOnly"` <b>`default`</b> to receive only SAS date values formatted, and all other data unformatted
```

Comment:
'datesOnly' <b>'default'</b> receives only SAS date values as formatted and all other data is unformatted.